showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -78,7 +78,11 @@ class ShowkaseProcessor @JvmOverloads constructor(
             ?.toSet()?.let { set.addAll(it) }
         return set
     }
-    override fun getSupportedOptions() = mutableSetOf("skipPrivatePreviews", "multiPreviewType")
+    override fun getSupportedOptions() = mutableSetOf(
+        "skipPrivatePreviews",
+        "skipFunctionsWithPreviewAnnotation",
```

Comment:
might consider a more explicit name, as this won't skip _all_ functions with preview annotations (ie, it keeps the ones with showkase annotation)
```suggestion
        "skipPreviewFunctionsWithoutShowkaseAnnotation",
```

or 
```suggestion
        "requireShowkaseAnnotation",
```

Reply:
this makes sense, thanks for the tip!